contents/gcmm.oscmeta

```diff
@@ -0,0 +1,20 @@
+{
+    "information": {
+        "name": "GCMM",
+        "author": "suloku",
+        "category": "utilities",
+        "peripherals": ["Wii Remote", "GameCube Controller", "SDHC"],
+        "version": "auto"
+    },
+    "source": {
+        "type": "github_release",
+        "format": "zip",
+        "repository": "suloku/gcmm",
+        "file": "gcmm_v*.zip"
```

Comment:
```suggestion
        "file": "gcmm_*.zip"
```
There is no "v" in the release file name

Reply:
no u